### DISCUSSION TITLE
goal-by-target in bootstrap5

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1614] [Q&A]
Hello, I'm back in the openSDG Business :-)

I did the update to 1.8.0 and set bootstrap5=true.

How do I get the goal-by-target layout back? 

I had copied this file: [https://github.com/open-sdg/open-sdg/blob/2.0.0-dev/_layouts/goal-by-target.html](https://github.com/open-sdg/open-sdg/blob/2.0.0-dev/_layouts/goal-by-target.html)

and put it into my _layouts folder[here](https://github.com/christoph-LSN/MT_Site/tree/develop/_layouts)

but it looks like the bootsrap5 goal page.

I also tried to put it into this folder [https://github.com/christoph-LSN/MT_Site/tree/develop/_includes/bootstrap5/components/goal](https://github.com/christoph-LSN/MT_Site/tree/develop/_includes/bootstrap5/components/goal)

but it still looks like the same....

[site_repo](https://github.com/christoph-LSN/MT_Site)

This is I want to have it:

![image](https://user-images.githubusercontent.com/94443931/168625638-f0aed0bd-ff63-4644-9a08-daeb3c7481b4.png)

Any ideas?

**Comment by brockfanning** [comment]
@christoph-LSN Good to hear from you. :) I know that we are planning to do a blog post with an example of how to override the goal layout in a way that recreates the goal-by-target layout, because we know that many implementations prefer this layout. Let's keep this open and I will update the discussion when that is posted.

**Comment by brockfanning** [comment]
@christoph-LSN In the meantime can I also ask for your feedback on the reasons for wanting the goal-by-target layout back? In particular, do you know if there are any shortcomings or flaws with the goal layout? Or is it purely a design preference? Or something else? Any feedback would be very helpful.

**Reply by christoph-LSN** [reply]
@brockfanning ha have not recognized any flaws yet.... It's purely a design preference and my customers likes the goal-by-target layout....

**Comment by christoph-LSN** [comment]
@brockfanning I have another thing i'd like to have: In my migration report data from different years are published. I'd like to have a field like the data_status complete with the year of the current data.

![image](https://user-images.githubusercontent.com/94443931/169996688-5908e65e-b701-4f65-ae57-5aed3dd61462.png)

Do you have an ides how do do this?
